clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -53,6 +53,9 @@ object ElasticsearchIndex {
   val Gvcf: ElasticsearchIndex[DocumentGvcf] =
     indexDocument[DocumentGvcf]
 
+  val WgsCram: ElasticsearchIndex[DocumentWgsCram] =
```

Comment:
Should these be moved into the model index classes? E.g. `GvcfIndex`?

Reply:
Right now they can't be, because the transfer-model project can't see the sources of the dataaccess-model project.